.github/workflows/testing.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install black
-        black -l 120 --exclude webtranslate/bottle.py --check webtranslate scripts/filter_update_langs scripts/lang_sync
+        black -l 120 --exclude webtranslate/bottle.py --check webtranslate scripts/filter_update_langs scripts/lang_sync scripts/eintsgit.py
```

Comment:
Do the scripts need individual listing? Or does "--check webtranslate scripts" also work?
Same in line 50.

Reply:
Issue is that 2 files do not have the `.py` extension, and will be skipped .. something to fix another day :D